OSError: LSAPI: Socket read/write error
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
[2024-05-14 06:04:55,766] ERROR in app: Exception on /verification [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 1938, in verification
    return render_template('verification.html',
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/loaders.py", line 115, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: verification.html
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 3>
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
[UID:2603][4175429] packetLen < 0
[UID:2603][4175429] packetLen < 0
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 4>
shared_instance is None
shared_instance: None
OSError: LSAPI: Socket read/write error
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 4>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 4>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 4>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 4>
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 4>
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 4>
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 4>
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
OSError: LSAPI: Socket read/write error
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 3>
OSError: LSAPI: Socket read/write error
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
[UID:2603][513348] packetLen < 0
[UID:2603][549021] packetLen < 0
[2024-05-14 13:25:34,441] ERROR in app: Exception on /verification [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 1905, in verification
    user_id = current_user.id
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'AnonymousUserMixin' object has no attribute 'id'
[2024-05-14 13:25:44,618] ERROR in app: Exception on /verification [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 1905, in verification
    user_id = current_user.id
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'AnonymousUserMixin' object has no attribute 'id'
shared_instance is None
shared_instance: None
[2024-05-14 14:32:50,113] ERROR in app: Exception on /verification [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 1905, in verification
    user_id = current_user.id
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'AnonymousUserMixin' object has no attribute 'id'
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 3>
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 3>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 3>
[UID:2603][4137265] packetLen < 0
[UID:2603][4137278] packetLen < 0
[UID:2603][4137278] packetLen < 0
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
[UID:2603][1466060] packetLen < 0
[UID:2603][1466061] packetLen < 0
[UID:2603][1466062] packetLen < 0
[UID:2603][1466062] packetLen < 0
[UID:2603][1544698] packetLen < 0
[UID:2603][1544698] packetLen < 0
[UID:2603][1544484] packetLen < 0
[UID:2603][1544698] packetLen < 0
[UID:2603][1544698] packetLen < 0
[UID:2603][1544484] packetLen < 0
[UID:2603][1544484] packetLen < 0
OSError: LSAPI: Socket read/write error
shared_instance is None
shared_instance: None
[UID:2603][1616143] packetLen < 0
[UID:2603][1616308] packetLen < 0
[UID:2603][1616308] packetLen < 0
[UID:2603][1616308] packetLen < 0
[UID:2603][1616308] packetLen < 0
[UID:2603][1616308] packetLen < 0
[2024-05-15 17:53:15,304] ERROR in app: Exception on /create_property [POST]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/getophfu/landbank_project/server.py", line 477, in create_property
    return render_template('createlisting.html')
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/getophfu/landbank_project/templates/createlisting.html", line 224, in top-level template code
    <a class="text-decoration-none" href="#">{{user.firstname}} {{user.lastname}}</a>
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 471, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'user' is undefined
[UID:2603][1616143] packetLen < 0
[2024-05-15 17:53:54,202] ERROR in app: Exception on /create_property [POST]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/getophfu/landbank_project/server.py", line 477, in create_property
    return render_template('createlisting.html')
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/getophfu/landbank_project/templates/createlisting.html", line 224, in top-level template code
    <a class="text-decoration-none" href="#">{{user.firstname}} {{user.lastname}}</a>
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 471, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'user' is undefined
[UID:2603][1616308] packetLen < 0
[UID:2603][1626219] packetLen < 0
[UID:2603][1626181] packetLen < 0
[UID:2603][1626219] packetLen < 0
[UID:2603][1626218] packetLen < 0
[UID:2603][1626181] packetLen < 0
[2024-05-15 17:59:07,734] ERROR in app: Exception on /create_property [POST]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/getophfu/landbank_project/server.py", line 477, in create_property
    return render_template('createlisting.html')
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/getophfu/landbank_project/templates/createlisting.html", line 224, in top-level template code
    <a class="text-decoration-none" href="#">{{user.firstname}} {{user.lastname}}</a>
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 471, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'user' is undefined
[UID:2603][1626219] packetLen < 0
[UID:2603][1649218] packetLen < 0
[UID:2603][1649218] packetLen < 0
[UID:2603][1649219] packetLen < 0
[UID:2603][1649218] packetLen < 0
[UID:2603][1649219] packetLen < 0
[2024-05-15 18:07:32,311] ERROR in app: Exception on /create_property [POST]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/getophfu/landbank_project/server.py", line 477, in create_property
    return render_template('createlisting.html')
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/getophfu/landbank_project/templates/createlisting.html", line 224, in top-level template code
    <a class="text-decoration-none" href="#">{{user.firstname}} {{user.lastname}}</a>
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 471, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'user' is undefined
[UID:2603][1649217] packetLen < 0
[UID:2603][1660196] packetLen < 0
[UID:2603][1660248] packetLen < 0
[UID:2603][1660196] packetLen < 0
[UID:2603][1660248] packetLen < 0
[UID:2603][1660196] packetLen < 0
[2024-05-15 18:12:28,030] ERROR in app: Exception on /create_property [POST]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/getophfu/landbank_project/server.py", line 477, in create_property
    return render_template('createlisting.html', user=user)
NameError: name 'user' is not defined
[UID:2603][1660196] packetLen < 0
[UID:2603][1679206] packetLen < 0
[UID:2603][1679206] packetLen < 0
[UID:2603][1679206] packetLen < 0
[UID:2603][1679206] packetLen < 0
[UID:2603][1679206] packetLen < 0
[UID:2603][1677241] packetLen < 0
[UID:2603][1727146] packetLen < 0
[UID:2603][1727311] packetLen < 0
[UID:2603][1739327] packetLen < 0
[UID:2603][1741180] packetLen < 0
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
[2024-05-18 00:38:24,240] ERROR in app: Exception on /signup [POST]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1276, in _execute_context
    self.dialect.do_execute(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/default.py", line 608, in do_execute
    cursor.execute(statement, parameters)
sqlite3.IntegrityError: UNIQUE constraint failed: user.email

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 1339, in signup
    db.session.commit()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/scoping.py", line 163, in do
    return getattr(self.registry(), name)(*args, **kwargs)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 1046, in commit
    self.transaction.commit()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 504, in commit
    self._prepare_impl()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 483, in _prepare_impl
    self.session.flush()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 2540, in flush
    self._flush(objects)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 2682, in _flush
    transaction.rollback(_capture_exception=True)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/util/langhelpers.py", line 68, in __exit__
    compat.raise_(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 2642, in _flush
    flush_context.execute()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/unitofwork.py", line 422, in execute
    rec.execute(self)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/unitofwork.py", line 586, in execute
    persistence.save_obj(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/persistence.py", line 239, in save_obj
    _emit_insert_statements(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/persistence.py", line 1135, in _emit_insert_statements
    result = cached_connections[connection].execute(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1011, in execute
    return meth(self, multiparams, params)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/sql/elements.py", line 298, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1124, in _execute_clauseelement
    ret = self._execute_context(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1316, in _execute_context
    self._handle_dbapi_exception(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1510, in _handle_dbapi_exception
    util.raise_(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1276, in _execute_context
    self.dialect.do_execute(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/default.py", line 608, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.IntegrityError: (sqlite3.IntegrityError) UNIQUE constraint failed: user.email
[SQL: INSERT INTO user (firstname, lastname, email, password, location, nin_number, nin_slip, bio, phone_number, phone_number_two, biz_name, biz_bio, biz_location, biz_number, cac_numbers, cac_slip, last_login_at, is_admin, profile_picture) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('Kelvin', 'Kelvin', 'kelvinibeh3@gmail.com', 'pbkdf2:sha256:150000$4E0DEgRB$20d657411d919459c1fd967545570caf95992a3ae52918d5de67162b46243230', None, None, None, None, None, None, None, None, None, None, None, None, '2024-05-18 04:38:24.238467', 0, None)]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 3>
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
OSError: LSAPI: Socket read/write error
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 3>
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
OSError: LSAPI: Socket read/write error
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 3>
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
[UID:2603][2822986] packetLen < 0
[UID:2603][2823000] packetLen < 0
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
OSError: LSAPI: Socket read/write error
OSError: LSAPI: Socket read/write error
OSError: LSAPI: Socket read/write error
shared_instance is None
shared_instance: None
[2024-05-24 07:03:04,703] ERROR in app: Exception on /signup [POST]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1276, in _execute_context
    self.dialect.do_execute(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/default.py", line 608, in do_execute
    cursor.execute(statement, parameters)
sqlite3.IntegrityError: UNIQUE constraint failed: user.email

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 1342, in signup
    db.session.commit()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/scoping.py", line 163, in do
    return getattr(self.registry(), name)(*args, **kwargs)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 1046, in commit
    self.transaction.commit()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 504, in commit
    self._prepare_impl()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 483, in _prepare_impl
    self.session.flush()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 2540, in flush
    self._flush(objects)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 2682, in _flush
    transaction.rollback(_capture_exception=True)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/util/langhelpers.py", line 68, in __exit__
    compat.raise_(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 2642, in _flush
    flush_context.execute()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/unitofwork.py", line 422, in execute
    rec.execute(self)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/unitofwork.py", line 586, in execute
    persistence.save_obj(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/persistence.py", line 239, in save_obj
    _emit_insert_statements(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/persistence.py", line 1135, in _emit_insert_statements
    result = cached_connections[connection].execute(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1011, in execute
    return meth(self, multiparams, params)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/sql/elements.py", line 298, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1124, in _execute_clauseelement
    ret = self._execute_context(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1316, in _execute_context
    self._handle_dbapi_exception(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1510, in _handle_dbapi_exception
    util.raise_(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1276, in _execute_context
    self.dialect.do_execute(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/default.py", line 608, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.IntegrityError: (sqlite3.IntegrityError) UNIQUE constraint failed: user.email
[SQL: INSERT INTO user (firstname, lastname, email, password, location, nin_number, nin_slip, bio, phone_number, phone_number_two, biz_name, biz_bio, biz_location, biz_number, cac_numbers, cac_slip, last_login_at, is_admin, profile_picture) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('chinenye', 'okpala', 'okpalanenyefranca@gmail.com', 'pbkdf2:sha256:150000$AtRjMRlm$496404b49f6a415aab09a36af904cf9f3deedb882d22ff28db9a8e69256df9ad', None, None, None, None, None, None, None, None, None, None, None, None, '2024-05-24 11:03:04.701426', 0, None)]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
[2024-05-24 07:05:31,274] ERROR in app: Exception on /signup [POST]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1276, in _execute_context
    self.dialect.do_execute(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/default.py", line 608, in do_execute
    cursor.execute(statement, parameters)
sqlite3.IntegrityError: UNIQUE constraint failed: user.email

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 1342, in signup
    db.session.commit()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/scoping.py", line 163, in do
    return getattr(self.registry(), name)(*args, **kwargs)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 1046, in commit
    self.transaction.commit()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 504, in commit
    self._prepare_impl()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 483, in _prepare_impl
    self.session.flush()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 2540, in flush
    self._flush(objects)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 2682, in _flush
    transaction.rollback(_capture_exception=True)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/util/langhelpers.py", line 68, in __exit__
    compat.raise_(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/session.py", line 2642, in _flush
    flush_context.execute()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/unitofwork.py", line 422, in execute
    rec.execute(self)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/unitofwork.py", line 586, in execute
    persistence.save_obj(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/persistence.py", line 239, in save_obj
    _emit_insert_statements(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/persistence.py", line 1135, in _emit_insert_statements
    result = cached_connections[connection].execute(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1011, in execute
    return meth(self, multiparams, params)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/sql/elements.py", line 298, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1124, in _execute_clauseelement
    ret = self._execute_context(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1316, in _execute_context
    self._handle_dbapi_exception(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1510, in _handle_dbapi_exception
    util.raise_(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1276, in _execute_context
    self.dialect.do_execute(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/default.py", line 608, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.IntegrityError: (sqlite3.IntegrityError) UNIQUE constraint failed: user.email
[SQL: INSERT INTO user (firstname, lastname, email, password, location, nin_number, nin_slip, bio, phone_number, phone_number_two, biz_name, biz_bio, biz_location, biz_number, cac_numbers, cac_slip, last_login_at, is_admin, profile_picture) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: ('chinenye', 'okpala', 'okpalanenyefranca@gmail.com', 'pbkdf2:sha256:150000$IVvhaegs$1f37403f7211e067fcd916959e50d534a217fd933473bbf0818ccaba56ea8803', None, None, None, None, None, None, None, None, None, None, None, None, '2024-05-24 11:05:31.273840', 0, None)]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
[UID:2603][2209477] packetLen < 0
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 11>
shared_instance is None
shared_instance: None
[UID:2603][3512107] packetLen < 0
[UID:2603][3512108] packetLen < 0
[UID:2603][3512108] packetLen < 0
[UID:2603][3512108] packetLen < 0
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 10>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 12>
shared_instance is None
shared_instance: None
[UID:2603][3562631] packetLen < 0
[UID:2603][3562631] packetLen < 0
[UID:2603][3562631] packetLen < 0
[UID:2603][3598555] packetLen < 0
[UID:2603][3598519] packetLen < 0
[UID:2603][3598555] packetLen < 0
[2024-05-26 00:30:46,655] ERROR in app: Exception on /settings [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1276, in _execute_context
    self.dialect.do_execute(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/default.py", line 608, in do_execute
    cursor.execute(statement, parameters)
sqlite3.OperationalError: no such column: user.city

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 2020, in settings
    return render_template('settings.html', properties=properties, approved_properties=approved_properties, pending_properties=pending_properties, user=user)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 136, in render_template
    ctx.app.update_template_context(context)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 838, in update_template_context
    context.update(func())
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/utils.py", line 368, in _user_context_processor
    return dict(current_user=_get_user())
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/utils.py", line 335, in _get_user
    current_app.login_manager._load_user()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/login_manager.py", line 359, in _load_user
    return self.reload_user()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/login_manager.py", line 321, in reload_user
    user = self.user_callback(user_id)
  File "/home/getophfu/landbank_project/server.py", line 1430, in load_user
    return User.query.get(int(user_id))
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/query.py", line 1018, in get
    return self._get_impl(ident, loading.load_on_pk_identity)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/query.py", line 1135, in _get_impl
    return db_load_fn(self, primary_key_identity)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/loading.py", line 286, in load_on_pk_identity
    return q.one()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/query.py", line 3490, in one
    ret = self.one_or_none()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/query.py", line 3459, in one_or_none
    ret = list(self)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/query.py", line 3535, in __iter__
    return self._execute_and_instances(context)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/query.py", line 3560, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1011, in execute
    return meth(self, multiparams, params)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/sql/elements.py", line 298, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1124, in _execute_clauseelement
    ret = self._execute_context(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1316, in _execute_context
    self._handle_dbapi_exception(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1510, in _handle_dbapi_exception
    util.raise_(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1276, in _execute_context
    self.dialect.do_execute(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/default.py", line 608, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such column: user.city
[SQL: SELECT user.id AS user_id, user.firstname AS user_firstname, user.lastname AS user_lastname, user.email AS user_email, user.password AS user_password, user.location AS user_location, user.nin_number AS user_nin_number, user.nin_slip AS user_nin_slip, user.bio AS user_bio, user.phone_number AS user_phone_number, user.phone_number_two AS user_phone_number_two, user.biz_name AS user_biz_name, user.biz_bio AS user_biz_bio, user.biz_location AS user_biz_location, user.biz_number AS user_biz_number, user.cac_numbers AS user_cac_numbers, user.cac_slip AS user_cac_slip, user.last_login_at AS user_last_login_at, user.city AS user_city, user.state AS user_state, user.street AS user_street, user.is_admin AS user_is_admin, user.profile_picture AS user_profile_picture 
FROM user 
WHERE user.id = ?]
[parameters: (9,)]
(Background on this error at: http://sqlalche.me/e/13/e3q8)
[2024-05-26 00:30:49,988] ERROR in app: Exception on /settings [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1276, in _execute_context
    self.dialect.do_execute(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/default.py", line 608, in do_execute
    cursor.execute(statement, parameters)
sqlite3.OperationalError: no such column: user.city

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 2020, in settings
    return render_template('settings.html', properties=properties, approved_properties=approved_properties, pending_properties=pending_properties, user=user)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 136, in render_template
    ctx.app.update_template_context(context)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 838, in update_template_context
    context.update(func())
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/utils.py", line 368, in _user_context_processor
    return dict(current_user=_get_user())
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/utils.py", line 335, in _get_user
    current_app.login_manager._load_user()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/login_manager.py", line 359, in _load_user
    return self.reload_user()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/login_manager.py", line 321, in reload_user
    user = self.user_callback(user_id)
  File "/home/getophfu/landbank_project/server.py", line 1430, in load_user
    return User.query.get(int(user_id))
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/query.py", line 1018, in get
    return self._get_impl(ident, loading.load_on_pk_identity)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/query.py", line 1135, in _get_impl
    return db_load_fn(self, primary_key_identity)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/loading.py", line 286, in load_on_pk_identity
    return q.one()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/query.py", line 3490, in one
    ret = self.one_or_none()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/query.py", line 3459, in one_or_none
    ret = list(self)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/query.py", line 3535, in __iter__
    return self._execute_and_instances(context)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/orm/query.py", line 3560, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1011, in execute
    return meth(self, multiparams, params)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/sql/elements.py", line 298, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1124, in _execute_clauseelement
    ret = self._execute_context(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1316, in _execute_context
    self._handle_dbapi_exception(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1510, in _handle_dbapi_exception
    util.raise_(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/base.py", line 1276, in _execute_context
    self.dialect.do_execute(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib64/python3.8/site-packages/sqlalchemy/engine/default.py", line 608, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such column: user.city
[SQL: SELECT user.id AS user_id, user.firstname AS user_firstname, user.lastname AS user_lastname, user.email AS user_email, user.password AS user_password, user.location AS user_location, user.nin_number AS user_nin_number, user.nin_slip AS user_nin_slip, user.bio AS user_bio, user.phone_number AS user_phone_number, user.phone_number_two AS user_phone_number_two, user.biz_name AS user_biz_name, user.biz_bio AS user_biz_bio, user.biz_location AS user_biz_location, user.biz_number AS user_biz_number, user.cac_numbers AS user_cac_numbers, user.cac_slip AS user_cac_slip, user.last_login_at AS user_last_login_at, user.city AS user_city, user.state AS user_state, user.street AS user_street, user.is_admin AS user_is_admin, user.profile_picture AS user_profile_picture 
FROM user 
WHERE user.id = ?]
[parameters: (9,)]
(Background on this error at: http://sqlalche.me/e/13/e3q8)
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 1>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 1>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 1>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 1>
[UID:2603][1730227] packetLen < 0
[UID:2603][1730250] packetLen < 0
[UID:2603][1730250] packetLen < 0
[UID:2603][1730227] packetLen < 0
[UID:2603][1730227] packetLen < 0
[UID:2603][1730227] packetLen < 0
[UID:2603][1730227] packetLen < 0
[UID:2603][1730251] packetLen < 0
[UID:2603][1769579] packetLen < 0
[UID:2603][1769579] packetLen < 0
[UID:2603][1769561] packetLen < 0
[UID:2603][1769579] packetLen < 0
[UID:2603][1769579] packetLen < 0
[UID:2603][1769578] packetLen < 0
[UID:2603][1769578] packetLen < 0
[UID:2603][1769561] packetLen < 0
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 2>
[UID:2603][1785737] packetLen < 0
[UID:2603][1785737] packetLen < 0
[UID:2603][1785738] [UID:2603][1785737] packetLen < 0
packetLen < 0
[UID:2603][1785739] packetLen < 0
[UID:2603][1785737] packetLen < 0
[UID:2603][1785738] packetLen < 0
[UID:2603][1785737] packetLen < 0
[UID:2603][1785738] packetLen < 0
[UID:2603][1785739] packetLen < 0
[UID:2603][1785739] packetLen < 0
[UID:2603][1785737] packetLen < 0
[UID:2603][1785737] packetLen < 0
[UID:2603][1785739] packetLen < 0
[UID:2603][1785739] packetLen < 0
[UID:2603][1785739] packetLen < 0
[UID:2603][1810163] packetLen < 0
[UID:2603][1810163] packetLen < 0
[UID:2603][1810163] packetLen < 0
[UID:2603][1810162] packetLen < 0
[UID:2603][1810163] packetLen < 0
[UID:2603][1810163] packetLen < 0
[UID:2603][1810163] packetLen < 0
[UID:2603][1855821] packetLen < 0
[UID:2603][1855821] packetLen < 0
[UID:2603][1855170] packetLen < 0
[UID:2603][1855486] packetLen < 0
[UID:2603][1855821] packetLen < 0
[UID:2603][1855170] packetLen < 0
[UID:2603][1855170] packetLen < 0
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 1>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 1>
shared_instance is None
shared_instance: None amenity: None properties: [] approved_properties: [] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 1>
[UID:2603][1881790] packetLen < 0
[UID:2603][1881791] packetLen < 0
[UID:2603][1881793] packetLen < 0
[UID:2603][1881795] packetLen < 0
[UID:2603][1881791] packetLen < 0
[UID:2603][1881793] packetLen < 0
[2024-05-26 01:55:50,093] ERROR in app: Exception on /approve_property/1 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/getophfu/landbank_project/server.py", line 976, in approve_property
    send_approval_notification(property_to_approve.user.email)
TypeError: send_approval_notification() missing 1 required positional argument: 'property_id'
shared_instance is None
shared_instance: None
[UID:2603][1894689] packetLen < 0
[UID:2603][1894690] packetLen < 0
[UID:2603][1893730] packetLen < 0
[UID:2603][1894689] packetLen < 0
[UID:2603][1894690] packetLen < 0
[UID:2603][1893730] packetLen < 0
[2024-05-26 02:02:54,364] ERROR in app: Exception on /approve_property/1 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/getophfu/landbank_project/server.py", line 976, in approve_property
    send_approval_notification(property_to_approve.user.email)
TypeError: send_approval_notification() missing 1 required positional argument: 'property_id'
Uploaded Image: house4.jpg
Uploaded Image: house3.jpg
Uploaded Image: house2.jpg
Uploaded Image: house1.jpg
Uploaded Document: simenons-mou.pdf
Uploaded Document: ekiti_cm_mou.pdf
shared_instance is None
shared_instance: None amenity: None properties: [<Property 1>] approved_properties: [] pending_properties: [<Property 1>] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 1>
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [<Property 1>] approved_properties: [<Property 1>] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 3>
[UID:2603][130343] packetLen < 0
[UID:2603][130343] packetLen < 0
[UID:2603][130343] packetLen < 0
[UID:2603][130343] packetLen < 0
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [<Property 1>] approved_properties: [<Property 1>] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 3>
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
[UID:2603][158155] packetLen < 0
[UID:2603][158155] packetLen < 0
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None
[UID:2603][213702] packetLen < 0
shared_instance is None
shared_instance: None
shared_instance is None
shared_instance: None amenity: None properties: [<Property 1>] approved_properties: [<Property 1>] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 4>
[UID:2603][227226] packetLen < 0
[UID:2603][227226] packetLen < 0
[UID:2603][227226] packetLen < 0
[UID:2603][256534] packetLen < 0
[UID:2603][256534] packetLen < 0
[UID:2603][256534] packetLen < 0
shared_instance is None
shared_instance: None amenity: None properties: [<Property 1>] approved_properties: [<Property 1>] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 1>
shared_instance is None
shared_instance: None amenity: None properties: [<Property 1>] approved_properties: [<Property 1>] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 4>
shared_instance is None
shared_instance: None amenity: None properties: [<Property 1>] approved_properties: [<Property 1>] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 4>
shared_instance is None
shared_instance: None amenity: None properties: [<Property 1>] approved_properties: [<Property 1>] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 1>
shared_instance is None
shared_instance: None amenity: None properties: [<Property 1>] approved_properties: [<Property 1>] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 1>
shared_instance is None
shared_instance: None amenity: None properties: [<Property 1>] approved_properties: [<Property 1>] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 1>
shared_instance is None
shared_instance: None amenity: None properties: [<Property 1>] approved_properties: [<Property 1>] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 1>
[UID:2603][286695] packetLen < 0
[UID:2603][286645] packetLen < 0
shared_instance is None
shared_instance: None amenity: None properties: [<Property 1>] approved_properties: [<Property 1>] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 1>
OSError: LSAPI: Socket read/write error
[UID:2603][291342] packetLen < 0
[UID:2603][291169] packetLen < 0
[UID:2603][291169] packetLen < 0
[UID:2603][309833] packetLen < 0
[UID:2603][309833] packetLen < 0
[UID:2603][309833] packetLen < 0
[UID:2603][307409] packetLen < 0
[UID:2603][307409] packetLen < 0
[UID:2603][309834] packetLen < 0
[UID:2603][309834] packetLen < 0
[UID:2603][307409] packetLen < 0
[UID:2603][307409] packetLen < 0
[UID:2603][340320] packetLen < 0
[UID:2603][340320] packetLen < 0
[UID:2603][340320] packetLen < 0
[UID:2603][340320] packetLen < 0
[UID:2603][340320] packetLen < 0
[UID:2603][340290] packetLen < 0
[2024-05-26 19:15:46,132] ERROR in app: Exception on /approve_property/2 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/getophfu/landbank_project/server.py", line 976, in approve_property
    send_approval_notification(property_to_approve.user.email)
TypeError: send_approval_notification() missing 1 required positional argument: 'property_id'
Uploaded Image: house4.jpg
Uploaded Image: house3.jpg
Uploaded Image: house2.jpg
Uploaded Image: house1.jpg
Uploaded Document: ekiti_cm_mou.pdf
Uploaded Document: Copy_of_Donation_to_Water_Factory_.pdf
shared_instance is None
shared_instance: None amenity: None properties: [<Property 1>, <Property 2>] approved_properties: [<Property 1>] pending_properties: [<Property 2>] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 1>
shared_instance is None
shared_instance: None amenity: None properties: [<Property 1>, <Property 2>] approved_properties: [<Property 1>, <Property 2>] pending_properties: [] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [] approved_shared: [] pending_shared: [] jva: [] approved_jva: [] pending_jva: [] user: <User 1>
Traceback (most recent call last):
  File "/home/getophfu/landbank_project/passenger_wsgi.py", line 1, in <module>
    from server import app as application 
  File "/home/getophfu/landbank_project/server.py", line 251
    shared_bathroom = db.Column(db.Boolean, nullable=True Fault=False)
                                                          ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/getophfu/landbank_project/passenger_wsgi.py", line 1, in <module>
    from server import app as application 
  File "/home/getophfu/landbank_project/server.py", line 251
    shared_bathroom = db.Column(db.Boolean, nullable=True Fault=False)
                                                          ^
SyntaxError: invalid syntax
OSError: LSAPI: Socket read/write error
[UID:2603][1237786] packetLen < 0
[UID:2603][1237786] packetLen < 0
[UID:2603][1237786] packetLen < 0
[UID:2603][1237810] packetLen < 0
[UID:2603][1237785] packetLen < 0
[UID:2603][1237786] packetLen < 0
[UID:2603][1237785] packetLen < 0
OSError: LSAPI: Socket read/write error
[UID:2603][1257630] packetLen < 0
[UID:2603][1257630] packetLen < 0
[UID:2603][1257604] packetLen < 0
[UID:2603][1289750] packetLen < 0
[UID:2603][1289766] packetLen < 0
[UID:2603][1289766] packetLen < 0
[UID:2603][1289750] packetLen < 0
[UID:2603][1318898] packetLen < 0
[UID:2603][1318839] packetLen < 0
[UID:2603][1318839] packetLen < 0
[UID:2603][1318898] packetLen < 0
[UID:2603][1318899] packetLen < 0
[UID:2603][1328424] packetLen < 0
[UID:2603][1328632] packetLen < 0
[UID:2603][1328424] packetLen < 0
[UID:2603][1328632] packetLen < 0
[UID:2603][1336587] packetLen < 0
[UID:2603][1336587] packetLen < 0
[UID:2603][1336564] packetLen < 0
[UID:2603][1336564] packetLen < 0
[UID:2603][1353270] packetLen < 0
[UID:2603][1353272] packetLen < 0
[UID:2603][1353274] packetLen < 0
[UID:2603][1353276] packetLen < 0
[UID:2603][1353274] packetLen < 0
[UID:2603][1353274] packetLen < 0
[UID:2603][1353274] packetLen < 0
[UID:2603][1353270] packetLen < 0
[UID:2603][1353270] packetLen < 0
[UID:2603][1353272] packetLen < 0
Uploaded Image: house4.jpg
Uploaded Image: house3.jpg
Uploaded Image: house2.jpg
Uploaded Image: house1.jpg
shared_instance: <Shared 1> amenity: Partly Furnished, Parking Space, Shared Kitchen, Shared Bathroom, Noise Level, Fitness Center, Internet Service properties: [<Property 1>] approved_properties: [] pending_properties: [<Property 1>] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [<Shared 1>] approved_shared: [] pending_shared: [<Shared 1>] jva: [] approved_jva: [] pending_jva: [] user: <User 1>
Uploaded Image: house4.jpg
Uploaded Image: house3.jpg
Uploaded Image: house2.jpg
Uploaded Image: house1.jpg
Uploaded Document: simenons-mou.pdf
Uploaded Document: ekiti_cm_mou.pdf
shared_instance: <Shared 1> amenity: Partly Furnished, Parking Space, Shared Kitchen, Shared Bathroom, Noise Level, Fitness Center, Internet Service properties: [<Property 1>, <Property 2>] approved_properties: [] pending_properties: [<Property 1>, <Property 2>] approved_shortlets: [] pending_shortlets: [] shortlet: [] shared: [<Shared 1>] approved_shared: [] pending_shared: [<Shared 1>] jva: [] approved_jva: [] pending_jva: [] user: <User 1>
[UID:2603][1365258] packetLen < 0
[UID:2603][1365259] packetLen < 0
[UID:2603][1365260] packetLen < 0
[2024-05-27 01:56:54,056] ERROR in app: Exception on /approve_property/1 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/getophfu/landbank_project/server.py", line 976, in approve_property
    send_approval_notification(property_to_approve.user.email)
TypeError: send_approval_notification() missing 1 required positional argument: 'property_id'
[2024-05-27 01:57:07,317] ERROR in app: Exception on /approve_property/2 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/getophfu/landbank_project/server.py", line 976, in approve_property
    send_approval_notification(property_to_approve.user.email)
TypeError: send_approval_notification() missing 1 required positional argument: 'property_id'
[2024-05-27 02:22:32,200] ERROR in app: Exception on /admin/shortlet_property/1 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 2045, in admin_shortlet_property_details
    shortlet_property = Shortlet.query.get_or_404(shortlet_id)
NameError: name 'Shortlet' is not defined
[2024-05-27 02:22:45,979] ERROR in app: Exception on /admin/shortlet_property/1 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 2045, in admin_shortlet_property_details
    shortlet_property = Shortlet.query.get_or_404(shortlet_id)
NameError: name 'Shortlet' is not defined
[2024-05-27 02:24:40,219] ERROR in app: Exception on /admin/shortlet_property/1 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 2045, in admin_shortlet_property_details
    shortlet_property = Shortlet.query.get_or_404(shortlet_id)
NameError: name 'Shortlet' is not defined
[2024-05-27 02:24:45,089] ERROR in app: Exception on /admin/shortlet_property/1 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 2045, in admin_shortlet_property_details
    shortlet_property = Shortlet.query.get_or_404(shortlet_id)
NameError: name 'Shortlet' is not defined
[2024-05-27 02:25:37,416] ERROR in app: Exception on /admin/shortlet/1 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 2045, in admin_shortlet_property_details
    shortlet_property = Shortlet.query.get_or_404(shortlet_id)
NameError: name 'Shortlet' is not defined
[2024-05-27 02:27:21,260] ERROR in app: Exception on /admin/shortlet/1 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 2047, in admin_shortlet_property_details
    return render_template('admin_shortlet_details.html', property=shortlet_property)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/getophfu/landbank_project/templates/admin_shortlet_details.html", line 66, in top-level template code
    <li><strong>Availability:</strong> {{ shortlet.shortlet_available }}</li>
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 471, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'shortlet' is undefined
[2024-05-27 02:28:53,320] ERROR in app: Exception on /admin/shortlet_property/1 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 2047, in admin_shortlet_property_details
    return render_template('admin_shortlet_details.html', property=shortlet_property)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/getophfu/landbank_project/templates/admin_shortlet_details.html", line 66, in top-level template code
    <li><strong>Availability:</strong> {{ shortlet.shortlet_available }}</li>
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 471, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'shortlet' is undefined
[2024-05-27 02:31:33,112] ERROR in app: Exception on /admin/shortlet_property/1 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 2047, in admin_shortlet_property_details
    return render_template('admin_shortlet_details.html', shortlet_property=shortlet_property)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/getophfu/landbank_project/templates/admin_shortlet_details.html", line 66, in top-level template code
    <li><strong>Availability:</strong> {{ shortlet.shortlet_available }}</li>
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 471, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'shortlet' is undefined
[2024-05-27 02:33:25,173] ERROR in app: Exception on /approve_shortlet/1 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/getophfu/landbank_project/server.py", line 998, in approve_shortlet
    send_approval_notification(shortlet_to_approve.user.email)
TypeError: send_approval_notification() missing 1 required positional argument: 'property_id'
[2024-05-27 02:54:33,602] ERROR in app: Exception on /approve_shared/1 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/getophfu/landbank_project/server.py", line 1019, in approve_shared
    send_approval_notification(shared_to_approve.user.email)
TypeError: send_approval_notification() missing 1 required positional argument: 'property_id'
shared_instance: <Shared 1>
shared_instance: <Shared 1>
[2024-05-27 03:01:40,268] ERROR in app: Exception on /cart [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/getophfu/landbank_project/server.py", line 1258, in cart
    return render_template('cart.html', cart_items=cart_items, image_files=image_files, user = user)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/loaders.py", line 115, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: cart.html
[2024-05-27 03:01:51,375] ERROR in app: Exception on /cart [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/getophfu/landbank_project/server.py", line 1258, in cart
    return render_template('cart.html', cart_items=cart_items, image_files=image_files, user = user)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/loaders.py", line 115, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: cart.html
shared_instance: <Shared 1>
shared_instance: <Shared 1>
shared_instance: <Shared 1>
shared_instance: <Shared 1>
shared_instance: <Shared 1>
OSError: LSAPI: Socket read/write error
shared_instance: <Shared 1>
shared_instance: <Shared 1>
shared_instance: <Shared 1>
[2024-05-28 03:41:23,793] ERROR in app: Exception on /approve_shared/1 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/getophfu/landbank_project/server.py", line 1019, in approve_shared
    send_approval_notification(shared_to_approve.user.email)
TypeError: send_approval_notification() missing 1 required positional argument: 'property_id'
shared_instance: <Shared 1>
shared_instance: <Shared 1>
shared_instance: <Shared 1>
shared_instance: <Shared 1>
shared_instance: <Shared 1>
shared_instance: <Shared 1>
shared_instance: <Shared 1>
shared_instance: <Shared 1>
shared_instance: <Shared 1>
shared_instance: <Shared 1>
shared_instance: <Shared 1>
shared_instance: <Shared 1>
shared_instance: <Shared 1>
shared_instance: <Shared 1>
shared_instance: <Shared 1>
shared_instance: <Shared 1>
[2024-05-28 04:59:20,661] ERROR in app: Exception on /property/details/1 [POST]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 1643, in property_details
    cart_item = CartItem(property_id=property_id, user_id=current_user.id)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'AnonymousUserMixin' object has no attribute 'id'
[2024-05-28 05:27:24,135] ERROR in app: Exception on /shared/details/1 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 1730, in shared_details
    return render_template('shared_details.html', shared=shared,image_files=shared_images, user = user)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/loaders.py", line 115, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: shared_details.html
[2024-05-28 05:27:31,147] ERROR in app: Exception on /shared/details/1 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 1730, in shared_details
    return render_template('shared_details.html', shared=shared,image_files=shared_images, user = user)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/loaders.py", line 115, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: shared_details.html
[2024-05-28 05:31:03,226] ERROR in app: Exception on /shared/details/1 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 1730, in shared_details
    return render_template('shared_details.html', shared=shared,image_files=shared_images, user = user)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/loaders.py", line 115, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: shared_details.html
[2024-05-28 05:31:28,300] ERROR in app: Exception on /shared/details/1 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 1730, in shared_details
    return render_template('shared_details.html', shared=shared,image_files=shared_images, user = user)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/loaders.py", line 115, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: shared_details.html
[2024-05-28 05:45:27,897] ERROR in app: Exception on /shared/details/1 [GET]
Traceback (most recent call last):
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/getophfu/landbank_project/server.py", line 1730, in shared_details
    return render_template('shared_details.html', shared=shared,image_files=shared_images, user = user)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/getophfu/landbank_project/templates/shared_details.html", line 425, in top-level template code
    {% if property.image_filenames %}
  File "/home/getophfu/virtualenv/landbank_project/3.8/lib/python3.8/site-packages/jinja2/environment.py", line 471, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'property' is undefined
